commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/CompareCheckboxTag.java

```diff
@@ -57,7 +57,7 @@ public int doStartTag() {
 				putValue("inCompare", true);
 			}
 			else {
-				putValue("inCompare", true);
```

Comment:
@alecsloan @ethanbustad  can we simplify :  `putValue("inCompare", cpDefinitionIds.contains(cpDefinitionId));`